modules/creating_a_blueprint.md

```diff
@@ -55,6 +55,7 @@ A JHipster blueprint:
 - is an NPM package, and is a Yeoman generator.
 - follows an extension of the Yeoman rules listed at [http://yeoman.io/generators/](http://yeoman.io/generators/) and can be installed, used and updated using the `yo` command. Instead of being prefixed by `generator-`, it is prefixed by `generator-jhipster-`, and instead of having just the `yeoman-generator` keyword, it must have 2 keywords, `yeoman-generator` and `jhipster-blueprint`.
 - A blueprint can only extend the following sub-generators (under the generators folder)
+    - app
```

Comment:
We can also extend `heroku`, `cypress`, `page`, `entities`, and `ci-cd` subgenerators 

`ci-cd` is only in `main` branch and not in the beta releases yet.

https://github.com/jhipster/generator-jhipster/search?p=1&q=extends+BaseBlueprintGenerator

Reply:
awesome, was not aware of that. I will add it to the documentation but also in the blueprint generator.

Reply:
done in both doc PR but I will add a bullet point in the main ticket